modules/add-contextual-data/tests/test_context_info_db.c

```diff
@@ -267,6 +289,34 @@ test_import_with_valid_csv(void)
 }
 
 static void
+test_import_with_valid_csv_crlf(void)
+{
+  gchar csv_content[] = "selector1,name1,value1\r\n"
```

Comment:
`fmemopen()` needs a non-const buffer in `_assert_import_csv_with_single_selector`.